editor/plugins/script_editor_plugin.cpp

```diff
@@ -989,10 +989,6 @@ void ScriptEditor::_res_saved_callback(const Ref<Resource> &p_res) {
 
 		RES script = se->get_edited_resource();
 
-		if (script->is_built_in()) {
-			continue; //internal script, who cares
-		}
-
```

Comment:
Second thoughts, rather then having a separate method and signal can we not handle it here?

Reply:
The `_res_saved_callback` isn't called for built-in resources. I removed this check, but it was useless anyways.